landoapi/treestatus.py

```diff
@@ -28,7 +28,7 @@ def is_open(self, tree):
         try:
             resp = self.get_trees(tree=tree)
         except TreeStatusError as exc:
-            if exc.status_code != 404:
+            if exc.status_code != 404 and exc.status_code != 400:
```

Comment:
nit: `if exc.status_code not in (400, 404):`

Only caught my eye because I'm used to using pylint, where this would trigger `consider-using-in (R1714)`.

Reply:
lol -- I missed this, will fix (here's a deja vu https://github.com/mozilla-conduit/lando-api/pull/97#discussion_r370308524)